chapter_16_javascript.asciidoc

```diff
@@ -637,7 +648,7 @@ on-failure messages into individual `expect/toBe` expressions.
 
 If you refresh the browser, you should see that all passes:
 
-
+// RITA: You don't usually have captions other than commits. Was this one intentional?
```

Comment:
yes, it's a slightly unusual one, because i'm using a code listing to display what's being shown in a web browser, not in a terminal.